### DISCUSSION TITLE
Inquiry Regarding Criteria for Family_ID Assignment in ABCD Study

### DISCUSSION THREAD

**Discussion by LinnBonaventure** [now-i-know-my-abcd/docs#66] [Where in the World is…? (Help)]
Hello ABCD community, 
I am currently utilizing the family_ID variable in the ABCD study to model shared environmental factors. However, I cannot seem to find the specific criteria used to define a 'family' within this context i.e.sharing a family_ID. Does it necessitate a genetic relation, shared surnames, cohabitation, or other criteria perhaps? 

I guess I could conduct some form of genetic similarity analyses to determine the extent of genetics shared among individuals within the same "family". Nevertheless it would be best to be able to document the actual definition of a family based on the actual ABCD criteria within the methods section of the paper. 

Thanks so much for any help here!
Cheers Linn

**Comment by laikevin128** [comment]
The Family ID (rel_family_id) is based on information obtained from guardians at enrollment and is not genetically informed. More information regarding Genetics can be found here https://wiki.abcdstudy.org/release-notes/non-imaging/genetics.html#genetic-principal-components-relatedness

**Comment by LinnBonaventure** [comment]
OK great, would you happen to know what specific information? Or perhaps
there are no hard critera for what is considered a family.

Thanks so much!
Kind regards Linn

**Comment by laikevin128** [comment]
There was no specific criteria.. thanks.

**Comment by LinnBonaventure** [comment]
OK great, thanks so much for the clarification!

Kind regards Linn
